joss/brachistochrone.py

```diff
@@ -0,0 +1,100 @@
+import numpy as np
```

Comment:
Kind of nervous that a file that people will be checking out in the future isn't tested at all, but I don't know a good way to test this, unless you build the model in a group.

Reply:
My only purpose here was to verify that the code that's in the paper was functional.  I can wrap it in a test case so that it will be tested when testflo is run from the top level.

The other option is to delete it, but i feel like the paper should be updated if the code changes.

Reply:
@Kenneth-T-Moore explicitly added a testflo call to the joss/test folder that tests the two documented example cases